packages/react-components/src/templates/EventPage.tsx

```diff
@@ -57,6 +59,7 @@ const EventPage: React.FC<EventPageProps> = ({
             {formatDistance(new Date(), new Date(lastModifiedDate))} ago
           </small>
         </Paragraph>
+        {props.speakers && <SpeakerList {...props} />}
```

Comment:
What happens if speakers is an empty array? I think the SpeakerList should be omitted